docs/migrating.rst

```diff
@@ -206,8 +206,6 @@ A list of these changes is enumerated below.
 +---------------------------------------+------------------------------------------------------------------------------+
 | ``Client.wait_until_login``           | Removed                                                                      |
 +---------------------------------------+------------------------------------------------------------------------------+
-| ``Client.messages``                   | Removed                                                                      |
```

Comment:
Why not just point to the new attribute in the table instead of adding it somewhere else?

Reply:
This changed didn't feel like it fit under the "models are stateful" heading so I moved it to the misc section